src/App.js

```diff
@@ -18,34 +21,80 @@ function App() {
 
   return (
     <AppWrapper>
-      {isEditFormShown ? 
-        <SaveGameForm 
-            onSubmit={addGameProfile} 
-            isEditFormShown={isEditFormShown} 
-            targetProfile={targetProfile} 
-            editGameProfile={editGameProfile}
-            cancelEditModus={cancelEditModus}/>
-      : 
-      <SaveGameFormWrapper>
-        <SaveGameForm onSubmit={addGameProfile} />
-        <GameCardsList 
-            savedGameProfiles={savedGameProfiles} 
-            onDelete={deleteGameProfile} 
-            onEdit={prepareEditModus}/>
-      </SaveGameFormWrapper>
-      }
+      <HeaderWrapper>
+        <Header/>
+      </HeaderWrapper>
+      <MainWrapper>
+        {isEditFormShown ? 
+          <SaveGameForm 
+              onSubmit={addGameProfile} 
+              isEditFormShown={isEditFormShown} 
+              targetProfile={targetProfile} 
+              editGameProfile={editGameProfile}
+              cancelEditModus={cancelEditModus}/>
+        : 
+        <SaveGameFormWrapper>
+          <SaveGameForm onSubmit={addGameProfile} />
+          <GameCardsList 
+              savedGameProfiles={savedGameProfiles} 
+              onDelete={deleteGameProfile} 
+              onEdit={prepareEditModus}/>
+        </SaveGameFormWrapper>
+        }
+      </MainWrapper>
+      <FooterWrapper>
+        <PlusIcon/>
+        <HomeIcon/>
+      </FooterWrapper>
     </AppWrapper>
   );
 }
 
 export default App;
 
 const AppWrapper = styled.div`
-  padding: 10px;
-  margin: 5px auto;
+  margin: 0 auto;
+  height: 100vh;
+  width: 100vw;
   max-width: 600px;
 `
+const HeaderWrapper = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  width: 100vw;
+  max-width: 1240px;
+  z-index: 10;
+`
+const MainWrapper = styled.main`
+  padding: 10px;
+  margin: 60px auto;
+  scrollbar-width: none;
+  &::-webkit-scrollbar {
+    display: none;
```

Comment:
have you tried ```overflow: hidden```?
cannot see your deployment so I'm not quiet sure here...

Reply:
It actually refers to the scrollbar and should implement that it is not shown. Does ``overflow:hidden`` work for this as well?

Reply:
as far as I know: yess :)